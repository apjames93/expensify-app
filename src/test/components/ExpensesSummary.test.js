import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('should rencer Expenses summary with 1 expense', () => {
    const wrapper = shallow(
        <ExpensesSummary 
            expenseCount={1} 
            expensesTotal={235}
        /> 
    );
    expect(wrapper).toMatchSnapshot();
});

test('should rencer Expenses summary with multiple expenses', () => {
    const wrapper = shallow(
        <ExpensesSummary 
            expenseCount={20} 
            expensesTotal={32165412346587546}
        /> 
    );
    expect(wrapper).toMatchSnapshot();

});