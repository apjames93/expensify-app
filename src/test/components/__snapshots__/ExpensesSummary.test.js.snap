// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should rencer Expenses summary with 1 expense 1`] = `
<div>
  <h1>
    viewing
    1
     
    expenses
    totalling
    $2.35
  </h1>
</div>
`;

exports[`should rencer Expenses summary with multiple expenses 1`] = `
<div>
  <h1>
    viewing
    20
     
    expenses
    totalling
    $321,654,123,465,875.44
  </h1>
</div>
`;
